TT-1618: Document how to use test tool

J-Lambo opened the pull request:
DO NOT MERGE - questions still to be answered. 

First draft of instructions for the Matching Service Test Tool. 

Author: @J-Lambo

J-Lambo committed:
TT-1618: Document how to use test tool

DO NOT MERGE - questions still to be answered. 

First draft of instructions for the Matching Service Test Tool. 

Author: @J-Lambo

richardTowers commented on README.md:
```diff
@@ -1,16 +1,73 @@
+
+This command will run a series of test scenarios. Successful results will return in green text. Failed results will return in red text with error messages.
+
+:question: Is this necessary to document?
```
Probably not - could just say "will run a series of test scenarios" and segue in to the next section.

richardTowers commented on README.md:
```diff
@@ -1,16 +1,73 @@
+
+| Scenario | Description |
+| -------- | ----------- |
+|          |             |
```
Looking at the output, we currently have these "scenarios" - they're not even unique at the moment, so I think they need a bit more descriptive names...

| Scenario | Description |
| -------- | ----------- |
| runForSimpleCase | |
| runForExtensiveCase | |
| runForWhenAllElementsAreVerifiedAndNoMultipleValues | |
| runForExtensiveCase | |
| runCase | |
| runForUserAccountCreation | |

richardTowers replied:
Added a separate PR to sort the descriptions of the scenarios out - #7

richardTowers replied:
| Scenario | Description |
| ----- | ----- |
|  Simple request with level of assurance 1 | |
|  Complex request with level of assurance 1 | |
|  Simple request with level of assurance 2 | |
|  Complex request with level of assurance 2 | |
|  Simple request with address missing optional fields | |
|  Simple user account creation request | |

richardTowers commented on README.md:
````diff
@@ -1,16 +1,73 @@
+  "verified": true
+```
+
+:question: Is this a clear enough example?
````
Yes, I think so.

J-Lambo committed:
BAU: Deleted questions

Removed prompt text for review and unnecessary guidance on error messages.

J-Lambo committed:
BAU: Add list of default test scenarios

Added list of newly-named test scenarios. 

Removed table with descriptions - need for additional description can be evaluated during user research.

J-Lambo commented:
@richardTowers Thanks for the speedy review. I've made a few changes based on your comments and added the list of test scenarios.
